packages/devtools_app/lib/src/screens/inspector_v2/inspector_controller.dart

```diff
@@ -642,35 +574,18 @@ class InspectorController extends DisposableController
     final pendingSelectionFuture = group.getSelection(
       selectedDiagnostic,
       treeType,
-      isSummaryTree: isSummaryTree,
+      isSummaryTree: true,
```

Comment:
can this param be removed?

Reply:
I made the default value for the parameter `false`, but we can't remove it because these inspector service methods are shared with the legacy inspector.